src/server/replica.cc

```diff
@@ -846,15 +847,15 @@ void DflyShardReplica::StableSyncDflyReadFb(Context* cntx) {
   DCHECK_GE(journal_rec_executed_, 1u);
   TransactionReader tx_reader{journal_rec_executed_.load(std::memory_order_relaxed) - 1};
 
-  if (master_context_.version > DflyVersion::VER0) {
-    acks_fb_ = fb2::Fiber("shard_acks", &DflyShardReplica::StableSyncDflyAcksFb, this, cntx);
-  }
+  acks_fb_ = fb2::Fiber("shard_acks", &DflyShardReplica::StableSyncDflyAcksFb, this, cntx);
```

Comment:
I guess the point is that we want to break replication with `VER0` so we don;t really care...

Reply:
yes, we stop supporting replication from DF 1.3 or older